Client/Frontend/Browser/BrowserViewController/BrowserViewController+URLBarDelegate.swift

```diff
@@ -109,6 +110,15 @@ extension BrowserViewController: URLBarDelegate {
         }
     }
 
+    func urlBarDidPressShoppingCart(_ urlBar: URLBarView, shoppingCart: UIButton) {
+        TelemetryWrapper.recordEvent(category: .action, method: .tap, object: .shoppingCartButton)
+        // TODO: Update closeButtonA11yLabel for the fakespot feature
+        let viewModel = BottomSheetViewModel(closeButtonA11yLabel: .CloseButtonTitle)
+        let bottomSheetVC = BottomSheetViewController(viewModel: viewModel,
+                                                      childViewController: FakespotBottomSheetChildViewController())
+        present(bottomSheetVC, animated: false)
```

Comment:
I think we need the animation here.

Reply:
animation set to true does not help, it makes it fade in, but if we let it to false and making the height half of the screen, it animates from the bottom to top.